test/access/manager/AccessManager.test.js

```diff
@@ -0,0 +1,523 @@
+const {
+  expectEvent,
+  expectRevert,
+  time: { duration },
+} = require('@openzeppelin/test-helpers');
+const { RestrictedMode } = require('../../helpers/enums');
+
+const AccessManager = artifacts.require('AccessManager');
+const AccessManagerAdapter = artifacts.require('AccessManagerAdapter');
+const AccessManaged = artifacts.require('$AccessManagedMock');
+
+const Ownable = artifacts.require('$Ownable');
+const AccessControl = artifacts.require('$AccessControl');
+
+const groupUtils = {
+  mask: group => 1n << BigInt(group),
+  decodeBitmap: hexBitmap => {
+    const m = BigInt(hexBitmap);
+    const allGroups = new Array(256).fill().map((_, i) => i.toString());
+    return allGroups.filter(i => (m & groupUtils.mask(i)) !== 0n);
```

Comment:
I don't see how this suggestion makes sense. We need `map` first because filtering from `new Array(256).fill()` will leave a shorter array without any indication of what indices were removed.